docs/reference/migration/migrate_7_9.asciidoc

```diff
@@ -67,5 +68,28 @@ setting. You may use `script.context.$CONTEXT.max_compilations_rate` for the par
 context.  For example, for the `processor_conditional` context, use
 `script.context.processor_conditional.max_compilations_rate`.
 
+====
+
+[discrete]
+[[breaking_79_settings_changes]]
+=== Settings changes
+
+[[deprecate_auto_import_dangling_indices]]
+.Automatically importing dangling indices is deprecated.
+
+[%collapsible]
+====
+*Details* +
+<<dangling-indices,Dangling indices>> will no longer be imported
```

Comment:
I think it'd help to move some things around here: IMO the `Details` section describes the change and the `Impact` tells the user what to do about it. How about this:

```diff
diff --git a/docs/reference/migration/migrate_7_9.asciidoc b/docs/reference/migration/migrate_7_9.asciidoc
index e60c4f3588d..acfd11fc38b 100644
--- a/docs/reference/migration/migrate_7_9.asciidoc
+++ b/docs/reference/migration/migrate_7_9.asciidoc
@@ -80,16 +80,18 @@ context.  For example, for the `processor_conditional` context, use
 [%collapsible]
 ====
 *Details* +
-<<dangling-indices,Dangling indices>> will no longer be imported
-automatically by default. A new
-<<dangling-indices-api,Dangling indices API>> is now available, which you
-can use to list, import, or delete. Because importing dangling indices
-automatically into the cluster is unsafe, use the dedicated API instead.
+Automatically importing <<dangling-indices,dangling indices>> into the cluster
+is unsafe and is now disabled by default. This feature will be removed entirely
+in {es} 8.0.0.

 *Impact* +
-Automatically importing dangling indices is now deprecated, disabled by
-default, and will be removed in (es) 8.0. If you need to re-enable
-auto-imports, set `gateway.auto_import_dangling_indices` to `true`.
+Use the <<dangling-indices-api,Dangling indices API>> to list, delete or import
+any dangling indices manually.
++
+Alternatively you can enable automatic imports of dangling indices, recovering
+the unsafe behaviour of earlier versions, by setting
+`gateway.auto_import_dangling_indices` to `true`. This setting is deprecated
+and will be removed in {es} 8.0.0. We do not recommend using this setting.

 ====
 //end::notable-breaking-changes[]
```